src/core/server/saved_objects/index.ts

```diff
@@ -22,3 +22,5 @@ export * from './service';
 export { SavedObjectsSchema } from './schema';
 
 export { SavedObjectsManagement } from './management';
+
+export { SavedObjectsSerializer, RawDoc as RawSavedObjectDoc } from './serialization';
```

Comment:
Can we rename `RawSavedObjectDoc` to `SavedObjectsRawDoc`. We've adopted a convention of all exports being prefixed by their service name.